pkg/apis/core/validation/validation_test.go

```diff
@@ -3034,15 +3034,15 @@ func TestValidatePersistentVolumeClaimUpdate(t *testing.T) {
 		},
 		"invalid-update-volume-attributes-class": {
 			oldClaim:                    validClaimVolumeAttributesClass1,
-			newClaim:                    validClaimNilVolumeAttributesClass,
+			newClaim:                    validClaimEmptyVolumeAttributesClass,
 			enableVolumeAttributesClass: true,
 			isExpectedFailure:           true,
 		},
 		"invalid-update-volume-attributes-class-to-nil": {
 			oldClaim:                    validClaimVolumeAttributesClass1,
 			newClaim:                    validClaimNilVolumeAttributesClass,
 			enableVolumeAttributesClass: true,
-			isExpectedFailure:           true,
+			isExpectedFailure:           false,
 		},
 		"invalid-update-volume-attributes-class-to-empty": {
```

Comment:
This looks like the same test case as line 3035 "invalid-update-volume-attributes-class"?